doc/pagination.md

```diff
@@ -0,0 +1,55 @@
+# Pagination
+
+`exo` uses cursor-based pagination for all of its APIs so that clients can page through results in a consistent and stateless way. The pagination scheme is described below.
+
+### Request parameters:
+
+| Parameter | required? | Description |
+| --------- | --------- | ----------- |
+| `cursor` | N | A cursor previously returned by another request, which represents a specific location in the collection |
+| `prev` | N | Maximum number of results to return before the cursor. In most cases, the default will be `0`. *Cannot be specified if `next` is specified*  |
+| `next` | N | Maximum number of results to return at or after the cursor. Each resource type will set a reasonable default, which is not guaranteed to be consistent between different resource types. *Cannot be specified if `prev` is specified* |
```

Comment:
s/resource type/collection/

also say that limit will be min(limit, maxLimit) - and actually make this true, if it is isn't